### ISSUE TITLE
Implement kaiju-mergeOutputs

### ISSUE THREAD

**Issue by artur-matysik** [nf-core/taxprofiler#288]
### Description of feature

Kaiju profiler offers `kaiju-mergeOutputs` function to merge Kaiju and Kraken (Bracken?) results  ([see in Kaiju documentation](https://github.com/bioinformatics-centre/kaiju#merging-outputs)). 
I managed to do that in a very dirty custom way, but it will be great to have this as a native option (+ result passed to taxpasta) if both profilers are enabled. 

Thanks for considering!

**Comment by jfy133** [comment]
Hi @artur-matysik thanks for the suggestion!

Hmm, I'm not sure about this. I don't really like/recommend the idea of merging taxonomic profiles of two different tools into one as they can represent very very different outputs. I think you have to be very very careful with this and should only do this in a very 'experimental' manner outside a 'production' pipeline (if that makes sense).

This would also violate the taxpasta structure I believe.

Thoughts @sofstam @Midnighter ?

**Comment by artur-matysik** [comment]
>This would also violate the taxpasta structure I believe. 

No, because output of merged Kaiju and Kraken results is in exact Kaiju format - it requires only KAIJU2TABLE (or similar, forgot the name of the process now) and can be feed directly to taxpasta (and its working fine, done it already for own analysis).

**Comment by jfy133** [comment]
Sorry - you're correct -  not structure, but rather concept.

By that I mean in taxpasta we want one table per tool, as there is then no-way to know (unless you embed this in sample/column names, but that's not something we can 'control' so again risks misleading people if not done properly). But like I said, lets see what the others say :)

**Comment by sofstam** [comment]
I think we can add the `kaiju-mergeOutputs` to taxprofiler as optional but I do not really like the idea to give it as input to taxpasta as it involves two profilers-if I understand it correctly.

**Comment by Midnighter** [comment]
I share @jfy133's concerns. I'm also wondering why/what for do you want this? Can you describe your use-case?
